app/assets/stylesheets/_forms.scss

```diff
@@ -70,21 +62,17 @@ textarea {
   resize: vertical;
 }
 
-input[type="search"] {
+[type="search"] {
```

Comment:
@joshuaogle I do agree that having `input` makes this more clear and easily scannable. The reason for the removal is that these declarations are overly qualified and cause overriding issues down the line in the cascade. For example, if we leave the `input` as is, the specificity is so high that you can’t override any of the styles here by just using a class: http://codepen.io/tysongach/pen/vLLzWp